### PR TITLE
fixes #1 Concordium Sign dApp

### LINKED ISSUE #1: Create a simple dApp to sign messages
Create a simple dApp that will be able to:

1. Receive a message via input
2. Sign it using the Concordium SDK: https://github.com/Concordium/concordium-node-sdk-js/tree/main/packages/common#sign-a-message
3. Return it in the browser as a base64 encoded string

It can be a simple javascript or reactjs app

### PR DESCRIPTION
**Pull request by vietredweb**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+  app:
+    image: aesirxio/concordium-sign-dapp:latest
+    environment:
+      REACT_APP_CLIENT_ID: ${REACT_APP_CLIENT_ID}
```
Comment (jatitoam) [inline docker-compose.yaml:7]: Do we need all these variables here?  I think that instead we need variables to communicate with a JSON-RPC Concordium server

**Review by jatitoam — CHANGES_REQUESTED** [review]

File: docker-compose.yaml (hunk 1 above)
Comment (vietredweb) [reply docker-compose.yaml]: I removed it @jatitoam , variables to communicate with JSON-RPC already in here https://github.com/vietredweb/concordium-sign-dapp/blob/main/src/config.ts

Comment (jatitoam) [reply docker-compose.yaml]: We need to remove exposure of those variables in these public repos please.  They are private nodes.

Please move them to `.env` files.

Comment (vietredweb) [reply docker-compose.yaml]: sure, I will move them to `.env`

**Review by lunguyenhat — APPROVED** [review]

**Pull request merged**